hack/lib/vars.bash

```diff
@@ -6,6 +6,10 @@ readonly BUILD_NUMBER=${BUILD_NUMBER:-$(uuidgen)}
 source "$(dirname "${BASH_SOURCE[0]}")/../../test/vendor/github.com/knative/test-infra/scripts/e2e-tests.sh"
 
 readonly KNATIVE_VERSION="${KNATIVE_VERSION:-v0.13.2}"
```

Comment:
Let's remove that parameter. It might be the same for serving & serving-operator, but not necessary. Also, for eventing & eventing operator.

Or, maybe us it as a default in:

```bash
eadonly KNATIVE_SERVING_VERSION="${KNATIVE_SERVING_VERSION:-${KNATIVE_VERSION}}"
```

Reply:
Remove that parameter? It's used in two places but they use a different prefix for branches (`release-` and `openshift-`).

Reply:
The second option won't work because we need those different prefixes in different places.

Reply:
What about:
```bash
readonly KNATIVE_VERSION="${KNATIVE_VERSION:-v0.13.2}"
readonly KNATIVE_SERVING_VERSION="${KNATIVE_SERVING_VERSION:-${KNATIVE_VERSION}}"
readonly KNATIVE_SERVING_REF="${KNATIVE_SERVING_REF:-release-${KNATIVE_SERVING_VERSION}}"
readonly KNATIVE_SERVING_OPERATOR_VERSION="${KNATIVE_SERVING_OPERATOR_VERSION:-${KNATIVE_VERSION}}"
readonly KNATIVE_SERVING_OPERATOR_REF="${KNATIVE_SERVING_OPERATOR_REF:-openshift-${KNATIVE_SERVING_OPERATOR_VERSION}}"
```

Then I'd add similar lines for eventing.

Reply:
And also, remove of parameters from `run_knative_serving_e2e_and_conformance_tests` and `run_knative_serving_operator_tests` functions.

Reply:
Also, we can merge it as it is. And, I will do those changes in my upcoming PR.

Reply:
I'm not sure. I'd probably prefer the minimum number of global variables. This solution works as well. It just moves some stuff from local functions to global vars.

Reply:
That would be required as the versions for serving and eventing are different. Also, if using other repos, that may need to use other ref then: `openshift-${KNATIVE_VERSION}` - this might be completely diffrent.